pandas/tseries/tests/test_base.py

```diff
@@ -1710,22 +1711,22 @@ def test_repeat(self):
     def test_nat(self):
         self.assertIs(pd.TimedeltaIndex._na_value, pd.NaT)
         self.assertIs(pd.TimedeltaIndex([])._na_value, pd.NaT)
-
+    
```

Comment:
@MattRijk If you look at the log of the failing travis build, at the bottom you will see the output of the linter:

```
pandas/tseries/tests/test_base.py:860:58: W291 trailing whitespace
pandas/tseries/tests/test_base.py:862:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:866:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:869:55: W291 trailing whitespace
pandas/tseries/tests/test_base.py:871:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:877:55: W291 trailing whitespace
pandas/tseries/tests/test_base.py:1713:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:1716:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:1721:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:1724:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:2710:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:2713:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:2718:1: W293 blank line contains whitespace
pandas/tseries/tests/test_base.py:2721:1: W293 blank line contains whitespace
```

You can normally set your editor to automatically remove trailing white space, then you would normally have less of such issues.